script/build_frontend

```diff
@@ -27,6 +27,10 @@ cp build/panels/*.html $OUTPUT_DIR/panels
 mkdir $OUTPUT_DIR_ES5/panels
 cp build-es5/panels/*.html $OUTPUT_DIR_ES5/panels
 
+# Chart code
```

Comment:
Don't we need to fingerprint it?

Reply:
I can't wait to Polymer 3 and native code splitting support via rollup/Webpack. No more hacky scripts on our part